docs/appendices/release-notes/unreleased.rst

```diff
@@ -209,9 +209,10 @@ Changes
 Fixes
 =====
 
-- Fixed the memory accounting of the circuit breaker for values which
-  types cannot be defined. Previously, the memory for such data types
-  was not accounted which could potentially lead to out of memory errors.
+- Improved the memory accounting for values of type ``geo_shape`` and
```

Comment:
do we need to add the undefined type here as well, such as we replaced the change log entry with the new one?